vars/artifactSetVersion.groovy

```diff
@@ -14,23 +15,67 @@ import groovy.text.SimpleTemplateEngine
 @Field Set GENERAL_CONFIG_KEYS = STEP_CONFIG_KEYS
 
 @Field Set STEP_CONFIG_KEYS = [
+    /** Defines the type of the artifact.
+    * @possibleValues `appContainer`
+    */
     'artifactType',
+    /** Defines the tool which is used for building the artifact.
+    * @possibleValues docker, dlang, golang, maven, mta, npm, pip, sbt
+    */
     'buildTool',
+    /** Controls if the changed version is committed and pushed to the git repository.
+    * If this is enabled (which is the default), you need to provide `gitCredentialsId` and `gitSshUrl`.
+    * @possibleValues `true`, `false`
+    */
     'commitVersion',
+    /** Specifies the source to be used for the main version which is used for generating the automatic version.
+    * * This can either be the version of the base image - as retrieved from the `FROM` statement within the Dockerfile, e.g. `FROM jenkins:2.46.2`
+    * * Alternatively the name of an environment variable defined in the Docker image can be used which contains the version number, e.g. `ENV MY_VERSION 1.2.3`
+    * * The third option `appVersion` applies only to the artifactType `appContainer`. Here the version of the app which is packaged into the container will be used as version for the container itself.
+    * @possibleValues FROM, (ENV name),appVersion
+    */
     'dockerVersionSource',
+    /** Defines a custom path to the descriptor file.*/
     'filePath',
+    /** Defines the ssh git credentials to be used for writing the tag.*/
     'gitSshKeyCredentialsId',
+    /** Allows to overwrite the global git setting 'user.email' available on your Jenkins server.*/
     'gitUserEMail',
+    /** Allows to overwrite the global git setting 'user.name' available on your Jenkins server.*/
     'gitUserName',
+    /** Defines the git ssh url to the source code repository.*/
     'gitSshUrl',
+    /** Defines the prefix which is used for the git tag which is written during the versioning run.*/
     'tagPrefix',
+    /** Defines the timestamp to be used in the automatic version string. You could overwrite the default behavior by explicitly setting this string.*/
     'timestamp',
+    /** */
```

Comment:
Information lost

Reply:
There is no information lost, there was no definition of this parameter. Could you add it?